src/hocs/ec-with-loading/ec-with-loading.js

```diff
@@ -0,0 +1,35 @@
+import { createHOC, createRenderFn } from 'vue-hoc';
+import EcLoading from '../../components/ec-loading';
+
+const withLoading = (Component, options = {}, renderOptions = {}) => createHOC(Component, {
+  ...options,
+  name: 'EcWithLoading',
+  functional: true,
+  props: {
+    isLoading: {
+      type: Boolean,
+      default: false,
+    },
+    isLoadingTransparent: {
+      type: Boolean,
+      default: true,
```

Comment:
That's true. However, if you don't set a default value for Boolean, it will become `false`. So it will not trigger the default value in EcLoading.

I was thinking to remove any constrains from HOC props and just let it pass as it is:
```
isLoading: {},
isLoadingTransparent: {},
loadingIconSize: {},
```

That would move the logic back to the EcLoading code and no code duplication is needed.